net/dnsdist/Makefile

```diff
@@ -66,6 +66,7 @@ CONFIGURE_ARGS+= \
 	--enable-libsodium \
 	--enable-protobuf \
 	--enable-re2 \
+	--with-lua=lua \
```

Comment:
Makes little sense since we already have luajit, it's a buildbot issue and nothing else.

Reply:
It will pick up luajit from the staging tree if it has been built before dnsdist.  It's a matter of choice.  Upstream prefers luajit by default, but it is failing for many platforms right now (`Requires GCC Multilib on host system to build!`).  I won't start a discussion of which is best, as I don't have a clue, but most of the packages are using lua now, and I would avoid having to install both.  On the other hand, this is a pretty big package.
```
$ egrep -c luajit/compile tmp/.packagedeps
2
$ egrep -c lua/compile tmp/.packagedeps
73
```